lucas_scheinkerman/README.md

```diff
@@ -1 +1,45 @@
-# Lucas Scheinkerman
+## Differential-drive robots
+
+A differential-drive robot (DDR from now) is a robot which has a two-wheel mechanism for allowing it to move. There is a third wheel which prevents the robot from falling and helps it balance, but which has not active role in the movement of the robot. br>
+This mechanism for moving the robot is based on the difference between the two wheels angular velocities, which is why it is called _differential_-drive.
+
+Before starting to describe how it works, it's important to notice that there are three velocities involved: The two velocities that correspond to the wheels and the velocity of the robot.
```

Comment:
If the third velocity is the velocity of the whole body, it might have two instead of one.